packages/docs/src/pages/readMe.md

```diff
@@ -136,3 +140,17 @@ curl -X GET 'localhost/meditor/api/me' \
 ```
 
 A user will be returned if your session is still active.
+
+#### Earthdata Login
+
+To log in through the mEditor API, you'll need to send a curl request to mEditor's login endpoint. The example below assumes that you have a `.netrc` file set up with your Earthdata Login (EDL) credentials. General information can be found on `.netrc` can be found in [cURL's documentation](https://everything.curl.dev/usingcurl/netrc); EDL-specific `.netrc` information can be found in the first part Earthdata Login's documentation on [How To Access Data With cURL And Wget](https://urs.earthdata.nasa.gov/documentation/for_users/data_access/curl_and_wget).
```

Comment:
The "General information" part seems to have a duplicated "can be found"

Reply:
Thanks for catching that! The file has been updated.